internal/pkg/manifest/workload_ecs_test.go

```diff
@@ -218,13 +218,28 @@ func TestSecret_UnmarshalYAML(t *testing.T) {
 			in:        "key: value",
 			wantedErr: errors.New(`cannot marshal "secret" field to a string or "secretsmanager" object`),
 		},
-		"should be able to unmarshal an SSM parameter name": {
-			in:     "/github/token",
-			wanted: Secret{from: aws.String("/github/token")},
+		"should be able to unmarshal an plain SSM parameter name": {
+			in: "/github/token",
+			wanted: Secret{from: stringOrFromCFN{
+				Plain: aws.String("/github/token"),
+			},
+			},
```

Comment:
Can we fix the `{` and `}` indentations?

```suggestion
			wanted: Secret{
			    from: stringOrFromCFN{
				Plain: aws.String("/github/token"),
			     },
			},
```